Scripts/Web_Scrappers/AmazonPriceTracker/tracker.py

```diff
@@ -0,0 +1,194 @@
+import time
+from selenium.webdriver.common.keys import Keys
+from amazon_config import (
+    get_web_driver_options,
+    get_chrome_web_driver,
+    set_ignore_certificate_error,
+    set_browser_as_incognito,
+    set_automation_as_head_less,
+    NAME,
+    CURRENCY,
+    FILTERS,
+    BASE_URL,
+    DIRECTORY
+)
+from selenium.common.exceptions import NoSuchElementException
+import json
+from datetime import datetime
```

Comment:
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.

you can use isort third party module to arranging your imports according to standards